distributions/standalone/assembly.xml

```diff
@@ -150,7 +150,6 @@
                 <include>org.mule.modules:mule-module-jbossts</include>
                 <include>org.mule.modules:mule-module-json</include>
                 <include>org.mule.modules:mule-module-management</include>
-                <include>org.mule.modules:mule-module-oauth</include>
```

Comment:
this removes the old module. The new extension is put in the distro the same way other extensions are